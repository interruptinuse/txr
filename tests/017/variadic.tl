(load "../common")

(with-dyn-lib (libc)
  (deffi printf-int-double "printf" int (str : int double)))

(printf-int-double "foo-%d = %4.3f\n" 123 4.56)
