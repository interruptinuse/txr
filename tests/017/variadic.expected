foo-123 = 4.560
